lib/properties.js

```diff
@@ -16,11 +17,10 @@ const Immutable = require('immutable');
  */
 const merge = (destination, source) => {
   // Ensure that the destination value is an Object. `== null ` covers both null and undefined
-  const dest = (destination == null ||
-  Object.getPrototypeOf(destination) !== Object.prototype) ? {} : destination;
+  const dest = isPlainObject(destination) ? destination : {};
 
   // Only merge source if it's an Object. `== null ` covers both null and undefined
```

Comment:
I don't think this comment applies any more, since we're using `isPlainObject` instead of `source == null`.